spec/data_formatter_spec.rb

```diff
@@ -0,0 +1,38 @@
+module MtgCli
+  describe DataFormatter do
+
+    describe '#reformat' do
+      let(:data) {
+        {
+          'NMS' => {
+            'name'  => 'Nemesis',
+            'code'  => 'NMS',
+            'cards' => [{'name' => 'Plains', 'type' => 'Land'},
+                        {'name' => 'Forest', 'type' => 'Land'}]
+            },
+          'RTR' => {
+            'name'  => 'Return to Ravnica',
+            'code'  => 'RTR',
+            'cards' => [{'name' => 'Forest', 'type' => 'Land', 'id' => '10'},
+                        {'name' => 'Simic Charm', 'type' => 'Instant'}]
+          }
+        }
+      }
+
+      it 'should add set data to card data' do
```

Comment:
For spec descriptions avoid 'should' or 'is expected to'. Use an active voice to describe the behavior under test.

~~~ruby
it 'adds set data to card data' do
...
end
~~~